.github/workflows/check-docs-pr-preview.yml

```diff
@@ -0,0 +1,47 @@
+# This workflow checks that PRs modifying website docs include a preview link in the PR description.
+# It enforces KubeStellar's documentation PR best practices.
+
+name: Check Docs PR Preview Link
+
+on:
+  pull_request:
+    types: [opened, edited, synchronize]
+    paths:
+      - 'docs/content/**'
+      - 'docs/README.md'
+      - 'docs/mkdocs.yml'
+      - 'docs/overrides/**'
+      - 'docs/scripts/**'
+      - 'docs/requirements.txt'
+      - 'docs/main.py'
+
+jobs:
+  check-preview-link:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v4
+      - name: Check for Preview Link in PR Description (bash)
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          set -e
+          PR_NUMBER=$(echo "${{ github.event.pull_request.number }}")
+          REPO="${{ github.repository }}"
+          BRANCH="${{ github.head_ref }}"
+          BODY=$(gh pr view $PR_NUMBER --repo $REPO --json body -q '.body')
+          DOC_BRANCH_REGEX='^doc-'
+          GITHUB_PAGES_URL="https://$(echo $REPO | cut -d'/' -f1).github.io/$(echo $REPO | cut -d'/' -f2)/$BRANCH"
+          PREVIEW_LINE=$(echo "$BODY" | grep -iE '^\s*Preview.*https?://')
+          if [ -n "$PREVIEW_LINE" ]; then
+            if [[ "$BRANCH" =~ $DOC_BRANCH_REGEX ]] && [[ "$PREVIEW_LINE" == *"$GITHUB_PAGES_URL"* ]]; then
+              echo "✅ Docs PR: doc- branch and valid preview link present."
+            elif [[ ! "$BRANCH" =~ $DOC_BRANCH_REGEX ]]; then
+              echo "::warning::⚠️ Docs PR: Preview link found, but branch does not start with doc-. Please consider using a doc- branch for automatic previews."
+            else
+              echo "::warning::⚠️ Docs PR: Preview link found, but it may not point to the expected GitHub Pages URL."
+            fi
+          else
+            echo "::error::❌ Docs PR: No preview link found. Please include a line starting with 'Preview' and a valid GitHub Pages URL for your changes."
+            exit 1
+          fi
```

Comment:
This will be less alarming if there is a terminal linebreak.
<img width="232" alt="image" src="https://github.com/user-attachments/assets/2fa99528-d080-49fd-89e8-1ca7a20d1c23" />

Reply:
There is still no terminal linebreak.